mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -320,10 +345,14 @@ def init_simple_endpoint_interfaces(self, worker_ip, spec):
             veth_peer = "veth-" + local_id
             veth = VethInterface(name=veth_name, peer=veth_peer)
 
+            pod_provider = PodProvider.K8S
+            if spec['type'] == 'arktos':
+                pod_provider = PodProvider.ARKTOS
+
             interfaces_list.append(Interface(
                 interface_id=interface_id,
                 interface_type=InterfaceType.veth,
-                pod_provider=PodProvider.K8S,
+                pod_provider=pod_provider,
```

Comment:
I know this is not introduced by your PR, just be curious why we need pod_provider.

Reply:
This is Arktos if the pod is created in artkos. The default is k8s